zeus/device/cpu/rapl.py

```diff
@@ -22,6 +22,7 @@
 import zeus.device.cpu.common as cpu_common
 from zeus.device.cpu.common import CpuDramMeasurement
 from zeus.device.exception import ZeusBaseCPUError
+from zeus.monitor.rapl import RaplMonitor
```

Comment:
The `zeus.device` module is the lowest-level in the module hierarchy, and other things (especially `zeus.monitor`) are supposed to build higher-level features using the abstractions provided by `zeus.device`. In that sense, putting `RaplMonitor` in `zeus.monitor` and importing it into `zeus.device.cpu` is the reverse direction. Instead, I think `RaplMonitor` should be directly inside `zeus/device/cpu/rapl.py`, since it's not really reused elsewhere.